docs/user/firststeps.rst

```diff
@@ -232,7 +232,7 @@ For example, if you want to keep the new notebook(s), run the following.
 .. code-block:: console
 
     git add notebooks # track everything inside the notebooks folder
-    git commit -m "Added some notebooks"
+    git commit -m "Added some notebooks" -a
```

Comment:
why?

Reply:
Well, I was forced to use it to make it advance, but now that I see it again, wait before merging:
* let's try reproduce it elsewhere, to rule out an artifact on my end - or is it prereq for the dir?
(interestingly, I didn't need to use it a bit later on in the 1st steps instructions, in another git op)

Reply:
what do you mean "to make it advance"? Are you sure you didn't forget the line above (`git add notebooks`)?

Reply:
@rokroskar : I'm sure I did not forget that line - in fact, that was a prerequisite for `-a` to work!